### ISSUE TITLE
Option to send a message to players not using the resource pack.

### ISSUE THREAD

**Issue by mibby** [Phoenix616/ResourcepacksPlugins#28]
@Phoenix616 Is it possible to add an option to send messages to players if they aren't using the resource pack?

**Comment by Phoenix616** [comment]
I assume you mean a message when they reject the pack? If so then that's a feature of the extended [ForceResourcepacks](https://www.spigotmc.org/resources/force-resourcepacks.10499/) version.

**Comment by mibby** [comment]
Not when they initially reject the pack but if they had already declined and never re-enable it. So you could inform them the pack isn't enabled on login.

**Comment by Phoenix616** [comment]
Yes, that's a functionality of the extended version. Initial rejects and subsequent pack requests look the exact same way to the server.
